e2e-tests/config/android.config.js

```diff
@@ -0,0 +1,20 @@
+const { getDeviceName, getPlatformVersion } = require('./common');
+const { config } = require('./wdio.conf');
+const { join } = require('path');
+
+config.capabilities = [
+  {
+    // The defaults you need to have in your config
+    platformName: 'Android',
+    maxInstances: 1,
+    // For W3C the appium capabilities need to have an extension prefix
+    // This is `appium:` for all Appium Capabilities which can be found here
+    // http://appium.io/docs/en/writing-running-appium/caps/
+    'appium:deviceName': getDeviceName('Android'),
+    'appium:platformVersion': getPlatformVersion('Android'),
+    'appium:automationName': 'UiAutomator2',
+    'appium:app': join(process.cwd(), './apps/em-devapp-3.2.5.apk'),
```

Comment:
I think we should discuss whether we want to run e2e tests against the devapp (in the 'serve' configuration) or against the actual e-mission app (after having built it in the 'build' configuration)

I can foresee challenges with trying to run tests through the devapp, and I also think that if we are aiming to test the true behavior of the app, the test target should be as realistic as possible (ie the fully built app)

@shankari What do you think?

Reply:
It would also negate the need to copy the devapp apk over to here.
When the app is built, the output apk appears somewhere in `platforms/android/...`